src/napari_plugin_manager/_tests/test_qt_plugin_dialog.py

```diff
@@ -347,7 +347,10 @@ def test_plugin_list_handle_action(plugin_dialog, qtbot):
                 InstallerActions.CANCEL,
                 version='3',
             )
-            mock.assert_called_with("", InstallerActions.CANCEL)
+            assert mock.call_count >= 2
+            assert mock.call_args_list[1] == call(
+                "cancelling...", InstallerActions.CANCEL
+            )
```

Comment:
The `set_bussy` is called twice when cancelling a job, but the second is done as a callback after the process is finished, that is indeterministic. And happen for test that assert failed,